index.js

```diff
@@ -1,10 +1,30 @@
 var express = require('express');
 var app = express();
+var bodyParser = require('body-parser');
 var port = 8000;
-
 // write the API here
 
+app.use(bodyParser.urlencoded({extended:true}));
+app.use(bodyParser.json());
+
+app.get('/api', function(req, res){
+  console.log('getting');
+  res.send({thoughts: 'My API here!'});
+});
+
+app.get('/api/data', function(req, res){
+  console.log("can you see?");
+  res.json({data: 'this is fake data'});
+});
+
+app.post('api/data', function(req, res){
+  console.log(req.body);
+  console.log(req.body.data);
 
+  var query = (req.body.data);
```

Comment:
This is the line you are having troubles with. It is a syntax issue with the way you are setting you var.